example/testlogger2.cpp

```diff
@@ -71,19 +71,29 @@ testlogger_cli::testlogger_cli(QObject *parent)
 
     loggerF1 = ul->createFileLogger("test", "log.txt", wc2);
     loggerF1->setModuleName("FILE1");
+    loggerF1->printThreadID(true);
     //loggerF2 = ul->createFileLogger("test", "log2.txt", wc4);

```

Comment:
would remove commented code

Reply:
it is useful depending on test to be done